src/transactions/feed/queryHelper.ts

```diff
@@ -350,6 +367,14 @@ export function handlePollResponse({
             shouldUpdateCachedTransactions = true
             hasNewCompletedTransaction = true
           }
+
+          if (
+            // Track cross-chain swap transaction status change to `Complete`
+            tx.__typename === 'CrossChainTokenExchange' &&
+            pendingStandbyTransactionHashes?.has(tx.transactionHash)
```

Comment:
it's also possible for the pending transaction to have come through blockchain-api only, without having a standby transaction (account restore) - we also need to check `knownPendingTransactionHashes`?

Reply:
ah yes, of course!